### PR TITLE
Update trainings list

### PR DESCRIPTION
**Pull request by ksuess**

### CODE REVIEW

**Review by tiberiuichim — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -145,12 +145,23 @@ Please create a new [issue](https://github.com/plone/volto/issues/new) or [pull
 
 ## Documentation
 
-You can find the (beta) documentation in [http://docs.voltocms.com](http://docs.voltocms.com)
+You can find the documentation in [http://docs.voltocms.com](http://docs.voltocms.com)
```
Comment (stevepiercy) [inline README.md:148]:
```suggestion
You can find the documentation in [https://docs.voltocms.com](https://docs.voltocms.com)
```

File: README.md [hunk 2]
```diff
@@ -145,12 +145,23 @@ Please create a new [issue](https://github.com/plone/volto/issues/new) or [pull
...
 
 ## Training
 
-A detailed training on how to create your own website using Volto is available
-as part of the Plone training docs at [https://training.plone.org/5/volto/index.html](https://training.plone.org/5/volto/index.html).
+On the [Plone Trainings Website](https://training.plone.org) you'll find
+Volto-dedicated open training materials plus React and other
+Javascript-centered trainings.
```
Comment (stevepiercy) [inline README.md:154]:
```suggestion
JavaScript-centered trainings.
```

File: README.md [hunk 3]
```diff
@@ -145,12 +145,23 @@ Please create a new [issue](https://github.com/plone/volto/issues/new) or [pull
...
 
-You can find the (beta) documentation in [http://docs.voltocms.com](http://docs.voltocms.com)
+You can find the documentation in [http://docs.voltocms.com](http://docs.voltocms.com)
 
 ## Training
 
-A detailed training on how to create your own website using Volto is available
-as part of the Plone training docs at [https://training.plone.org/5/volto/index.html](https://training.plone.org/5/volto/index.html).
+On the [Plone Trainings Website](https://training.plone.org) you'll find
+Volto-dedicated open training materials plus React and other
```
Comment (stevepiercy) [inline README.md:153]:
```suggestion
Volto-dedicated open training materials, plus React and other
```

File: README.md [hunk 4]
```diff
@@ -145,12 +145,23 @@ Please create a new [issue](https://github.com/plone/volto/issues/new) or [pull
...
+
+- [Mastering Plone 6 Development](https://training.plone.org/5/mastering-plone/)
+  The comprehensive training on Plone 6 with best practice tips for developers and integrators.
+- [Volto](https://training.plone.org/5/volto/index.html)
+  A detailed training on how to create your own website using Volto frontend.
+- [Volto Hands-On](https://training.plone.org/5/voltohandson/index.html)
+- [Volto Add-ons Development](https://training.plone.org/5/voltoaddons/index.html)
+- [Plone Deployment](https://training.plone.org/5/plone-deployment/index.html)
+- [React](https://training.plone.org/5/react/index.html)
+- [JavaScript For Plone Developers](https://training.plone.org/5/javascript/index.html)
```
Comment (stevepiercy) [inline README.md:165]: Does MkDocs have something like Intersphinx? MkDocStrings does.

https://mkdocstrings.github.io/usage/#cross-references-to-other-projects-inventories

Comment (stevepiercy) [reply README.md:165]: Also is there an `include` feature in MkDocs? That would be nicer than copy-paste the list of links between the `README.md` and `docs/source/getting-started/others.md`.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor corrections, and a question about MkDocs usage. I looked in the docs, but couldn't find answers.

File: README.md (hunk 4 above)
Comment (ksuess) [reply README.md:165]: Yes, avoiding redundance would be good. 
I just leave it like this, as the documentation should be updated if necessary, but will soon be moved to doc.plone.org in a format that probably will change.

**Review by stevepiercy — APPROVED** [review]

**Pull request merged**